packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/EthERC20Bridge.sol

```diff
@@ -101,16 +102,27 @@ contract EthERC20Bridge is L1Buddy {
      */
     function notifyCustomToken(
         address l1Address,
+        uint256 maxSubmissionCost,
         uint256 maxGas,
         uint256 gasPriceBid
-    ) external payable {
+    ) external payable returns (uint256) {
         address l2Address = customL2Tokens[l1Address];
         require(l2Address != address(0), "NOT_REGISTERED");
-        sendPairedContractTransaction(
-            maxGas,
-            gasPriceBid,
-            abi.encodeWithSignature("customTokenRegistered(address,address)", l1Address, l2Address)
-        );
+        return
```

Comment:
I think we want an event emitted here including the msg num from the `createRetryableTicket` call. `ActivatedCustomToken` or something like that